Failing test: `test_valid_signature_id[Valid signature ID - Rule is not ignored` in Analysisd integration tests (Tier 0/1)

## Description

The following integration test related to `if_sid` rule evaluation in `analysisd` is currently failing and should be investigated:

### Failing test:
```
test_analysisd/test_signature_id/test_valid_signature_id.py::test_valid_signature_id[Valid signature ID - Rule is not ignored]
```

### Affected versions: 
- 4.12.1
- 4.13.0
- 4.14.0

### Error summary:

The test is intended to verify that when a rule includes a valid `if_sid` referencing an existing rule ID, no warning messages about invalid or missing signature IDs should appear in the log.

However, the test fails due to unexpected log messages indicating that the referenced SIDs were **not found**:

```python
assert not log_monitor.callback_result
E       AssertionError: assert not ('99905', '99906')
E        +  where ('99905', '99906') = <wazuh_testing.tools.monitors.file_monitor.FileMonitor object at 0x7f491ebab730>.callback_result
```

This indicates that the custom rules containing those signature IDs may not be properly loaded or parsed before the test executes.

No logs like these should be found when the test runs:
- `Signature ID '(...)' was not found and will be ignored in the 'if_sid'.`
- `Empty 'if_sid' value. Rule '(...)' will be ignored.`

### Notes

All other tests in the suite passed:

```
========== 1 failed, 84 passed, 57142 deselected ==========
```

This suggests the issue is isolated to the `test_valid_signature_id.py` test file or its related rule configuration logic.

### Workflow run

- [GitHub Actions Run](https://github.com/wazuh/wazuh/actions/runs/15303506472/job/43050113548)

Please investigate the rule loading mechanism and verify that rules `99905` and `99906` are available and valid at the time `analysisd` starts.

# Update

## Problem Description

The error occurs because the test attempts to load rule `99906`, but it fails due to missing rule `99905`.  
Here’s the assertion that fails:

```
assert not log_monitor.callback_result
E       AssertionError: assert not ('99905', '99906')
E        +  where ('99905', '99906') = <wazuh_testing.tools.monitors.file_monitor.FileMonitor object at 0x7f491ebab730>.callback_result
```

These rules were introduced in the following PR:  
https://github.com/wazuh/wazuh/pull/29653

Even after replicating the environment, the issue persists. Tested locally by installing Wazuh `4.12.1` and running the test using Python `3.12`:

```bash
sudo pytest tests/integration/test_analysisd/test_signature_id/test_valid_signature_id.py
```

✅ The test passes:

```
tests/integration/test_analysisd/test_signature_id/test_valid_signature_id.py . [100%]
```

⚠️ However, several deprecation warnings are shown during test execution (see full output in logs).

---

## QA Testing Notes

To replicate the test environment:

1. QA team uses **Python 3.12**
2. Install **Wazuh 4.12.1**
3. Clone the QA repository
4. Set up the virtual environment:

```bash
python3.12 -m venv wazuh_env_py312
source wazuh_env_py312/bin/activate
pip install --upgrade pip setuptools wheel
pip install ./qa-integration-framework/
cd /home/vagrant/wazuh/
pytest tests/integration/test_analysisd/test_signature_id/test_valid_signature_id.py
```

---

## Next Steps

It seems the failure is related to the CBList for those rules not being properly loaded, which prevents rule `99906` from being recognized.

We should consider modifying the workflow to output more detailed information from `ossec.log`, in order to confirm whether the rule is not being loaded due to this issue.

--- a/src/wazuh_testing/data/configuration_template/all_disabled_ossec.conf
+++ b/src/wazuh_testing/data/configuration_template/all_disabled_ossec.conf
@@ -55,6 +55,9 @@
     <list>etc/lists/audit-keys</list>
     <list>etc/lists/amazon/aws-eventnames</list>
     <list>etc/lists/security-eventchannel</list>
+    <list>etc/lists/malicious-ioc/malware-hashes</list>
+    <list>etc/lists/malicious-ioc/malicious-ip</list>
+    <list>etc/lists/malicious-ioc/malicious-domains</list>
 
     <!-- User-defined ruleset -->
     <decoder_dir>etc/decoders</decoder_dir>

